controller/src/beerocks/master/db/db.cpp

```diff
@@ -20,12 +20,12 @@ using namespace beerocks_message;
 using namespace son;
 using namespace net;
 
-static const std::string timestamp_str            = "timestamp";
-static const std::string timelife_delay_str       = "timelife";
-static const std::string initial_radio_enable_str = "initial_radio_enable";
-static const std::string initial_radio_str        = "initial_radio";
-static const std::string selected_band_enable_str = "selected_band_enable";
-static const std::string selected_bands_str       = "selected_bands";
+const std::string db::TIMESTAMP_STR            = "timestamp";
+const std::string db::TIMELIFE_DELAY_STR       = "timelife";
+const std::string db::INITIAL_RADIO_ENABLE_STR = "initial_radio_enable";
```

Comment:
nitpick:
string are strings: underscore is not different from space in any aspect, so why go "technical", why not simply use natural language:
```suggestion
const std::string db::INITIAL_RADIO_ENABLE_STR = "initial radio enable";
```

Reply:
the strings are used as parameter-key-names (aka options in UCI). although the BPL could have restructured key-names from/to the suggested strings I preferred not to introduce this translation - as there is no added value to it.